src/ui/panels/quests.js

```diff
@@ -0,0 +1,243 @@
+        });
+        this.dashboard.scene.add(
+            this.questsScrollWindow.name,
+            this.questsScrollWindow,
+            true
+        );
+        this.refreshQuests();
+
+        this.statsScrollWindow = new ScrollWindow({
+            name: "stats",
+            x: 526,
+            y: 251,
+            textStartOffsetY: 5,
+            width: 190,
+            height: 214,
+            numColumns: 1,
+            padding: 3,
+        });
+
+        this.dashboard.scene.add(
+            this.statsScrollWindow.name,
+            this.statsScrollWindow,
+            true
+        );
+        this.refreshStats();
+
+        this.currentTab = CONSTANTS.QUEST_TABS.QUESTS;
+        this.openTab(CONSTANTS.QUEST_TABS.QUESTS);
+
+        // Default to hidden
+        this.setVisible(false);
+
+        // Destructor
+        dashboard.events.once("shutdown", () => this.destroy());
+    }
+
+    // TODO: handle other tabs?
+    openTab(tab) {
+        this.closeCurrentTab();
+        switch (tab) {
+            case CONSTANTS.QUEST_TABS.QUESTS:
+                this.questsTabButton.setAlpha(1);
+                this.isQuestTextVisible = true;
+                this.refreshQuests();
+                break;
+
+            case CONSTANTS.QUEST_TABS.STATS:
+                this.statsTabButton.setAlpha(1);
+                this.isStatTextVisible = true;
+                this.refreshStats();
+                break;
+
+            default:
+                console.log("Error: invalid quest tab.");
+        }
+
+        this.currentTab = tab;
+    }
+
+    // TODO: handle other tabs?
+    closeCurrentTab() {
+        switch (this.currentTab) {
+            case CONSTANTS.QUEST_TABS.QUESTS:
+                this.questsTabButton.setAlpha(0.01);
+                this.questsScrollWindow.setVisible(false);
+                this.isQuestTextVisible = false;
+                break;
+
+            case CONSTANTS.QUEST_TABS.STATS:
+                this.statsTabButton.setAlpha(0.01);
+                this.statsScrollWindow.setVisible(false);
+                this.isStatTextVisible = false;
+                break;
+
+            default:
+                console.log("Error: attempting to close invalid quest tab.");
+        }
+    }
+
+    refreshStats() {
+        this.statsScrollWindow.clearObjects();
+        let statTexts = this.dashboard.scene.get(CONSTANTS.SCENES.STATS).getStats();
+
+        for (const statkey in statTexts) {
+            let row = new TextRow(this.statsScrollWindow, 0, 0, []);
+
+            let statText = this.statsScrollWindow.add
+                .text(0, 0, statTexts[statkey]["text"], {
+                    font: "14px runescape white",
+                })
+                .setDepth(3);
+            row.add(statText);
+
+            let statTxtAmount = this.statsScrollWindow.add
+                .text(100, 0, getItemText(statTexts[statkey]["amount"])[0], {
+                    font: "14px runescape white",
```

Comment:
```js
font: "16px runescape",
fill: "orange",
```

Same thing here, easier to read with the numbers orange imo, matches the right click menu

Reply:
updated: https://github.com/EthanVieira/rs-clicker/pull/63/commits/294623b39a2c655d7ba30fc8181b48fd4e8f8a2c